### DISCUSSION TITLE
Library app doesn't match prototype

### DISCUSSION THREAD

**Discussion by JoshFialkoff** [Budibase/budibase#3253] [Help]
I am creating my first application using https://budibase.com/blog/crud-app/ as a model.
Mine doesn't match the video.
Is it possible to download the prototype?

**Comment by mjashanks** [comment]
Hi Josh. That blog post is a little out of date. Try this tutorial: https://docs.budibase.com/quickstart-tutorials/crud-app-with-budibase-db
